### DISCUSSION TITLE
2022-09-14 Weekly Meeting 73 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#293] [Weekly Meetings]
Weekly Meeting 73 is happening on Wednesday September 14th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers
- UX vote #222 approved and the repo is formed. 

### Working Group Updates:
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [OMI Website update](https://github.com/omigroup/website)
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research: TBD](https://github.com/omigroup/ux-research) (setting up the repo) 
- any others?

### Discussions: 
- #289 with Jesse 
- #285 with Jin 
- new protocol suggestions? What are you working on or have you seen that we should be setting our eyes on next? 

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/media/issues/49
Have something interesting to promote? 
We want to create some playlists on our youtube/channels that includes OMI member content to feature too. Please send any of those over to the OMI/media channel :)

**Comment by mrmetaverse** [comment]
# Website 
- reviewing current state

**Comment by mrmetaverse** [comment]
# MSF 
- [Notes](https://github.com/omigroup/MSF-Delegates/discussions/24)
- general meeting today with MSF 
  - exploratory groups are to create a charter, to cover _what_ topics we will be looking into. 
  - some other updates from the 3d asset interop exploratory group 
  - there is a new exploratory group "digital asset group" building concensus and drafting proposal for existing and emerging standards for digital rights, portability, etc. 
  - also a digital fashion wearables exploratory proposal (clothing, layering, shoes, hats, accessories) - there are a lot of things considered wearables, but the non-humanoid avatars, base, head customization, rigging, and loot will be a future group. 
  - proposal for real and virtual world integration to build consensus, the scope of whcih includes defining constructs describing the physical world, reality, and technical methods to integrate digital content with physical world. 
- biggest action for the communithy now: 
  - please review and comment on the [exploratory proposals](https://github.com/omigroup/MSF-Delegates/discussions/24)

**Comment by mrmetaverse** [comment]
# Dework demo 
- please check out dework.xyz 
- related to numerous tix #285 
- https://github.com/omigroup/media/issues/49
- maybe we get Philip in here to talk about [lovemachine](https://www.inc.com/magazine/20110301/philip-rosedale-on-freelancing-business-processes.html)
